### PR TITLE
Feature: Tutorial text

### LINKED ISSUE #13: Feature: Tutorial Text
Criteria:
- Have a Text object in the scene that will display any arbitrary text to be used to show tutorial texts.

### PR DESCRIPTION
**Pull request by ThijsvanDijk**
## Description
Introduced a simple prefab for displaying tutorial texts. It is possible to update the text from an event. It's also possible to hide the text from an event

## Setting up testing environment
1. In the project window navigate to Project/Assets/Scenes and open the TutorialText scene.
2. Press Play.

#### Testing the Tutorial Text
Criteria:
- [ ] Text can update on demand
- [ ] Text can hide and show on demand    

## Checklist
_These checks should always be true for your pull request_
- [x] My pull request is for one story/feature.
- [x] Every individual commit in this pull request is logical.
- [x] All code, documentation, commits and player seen texts are in English.
- [x] I have deleted unused / unnecessary code.
- [x] If my edits need a change in documentation, then I have updated the documentation.
- [x] All code is in correspondence with the code conventions.

### CODE REVIEW

File: Assets/Scripts/UI/TutorialTest.cs [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+        if (updateNeeded)
+        {
+            updateTutorialText.Invoke($"Time since startup: {(int)Time.realtimeSinceStartup}");
+        }        
+
+        if(Time.realtimeSinceStartup > 10f && updateNeeded)
+        {
+            tutorialOver.Invoke();
+            updateNeeded = false;
+        }
```
Comment (Jeroenvn) [inline Assets/Scripts/UI/TutorialTest.cs]: Update should only contain method calls or one line code

File: Assets/Scripts/UI/TutorialText.cs [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    private void Start()
+    {
+        text = GetComponent<TMP_Text>();
+    }
+
+    /// <summary>
+    /// Updates the text displayed by the Text object.
+    /// Call this function on an event.
+    /// </summary>
+    /// <param name="text">The new text for the </param>
```
Comment (Jeroenvn) [inline Assets/Scripts/UI/TutorialText.cs]: "The new text for the ...." 
For the what?

**Review by Jeroenvn — CHANGES_REQUESTED** [review]
There are some code conventions that are broken.
I was also unable to see any text on screen.

File: Assets/Scripts/UI/TutorialTest.cs (hunk 1 above)
Comment (ThijsvanDijk) [reply Assets/Scripts/UI/TutorialTest.cs]: Update now only contains call to UpdateText()

File: Assets/Scripts/UI/TutorialText.cs (hunk 2 above)
Comment (ThijsvanDijk) [reply Assets/Scripts/UI/TutorialText.cs]: Fixed comment

**Review by Jeroenvn — APPROVED** [review]
Looks good!
gif

**Pull request merged**